js/services/time.js

```diff
@@ -0,0 +1,50 @@
+  (function() {
+    'use strict';
+
+    // Gets time in 12-hour format
+    function getTime(timeFormat) {
+      var currentTime = new Date();
+      var hours = currentTime.getHours();
+      var minutes = currentTime.getMinutes();
+      var time = "The time is ";
+
+      // 24 hour
+      if (timeFormat == "24") {
+        time += hours + ":" + minutes;
+      }
+
+      // 12 hour
+      else {
+        if (minutes < 10) {
+          minutes = "0" + minutes;
+        }
+
+        else if (hours > 11) {
+          if (hours != 12) {
+            hours = hours - 12;
+          }
+          time += hours + ":" + minutes + " " + "PM";
+        }
+
+        else { // hours <= 11
+          time += hours + ":" + minutes + " " + "AM";
+        }
+      }
+
+      return time;
+    }
+
+
+    function TimeService() {
+        var service = {};
+        var say = require('say');
```

Comment:
You never actually use `vocal` just `say`, which brings in unnecessary dependencies.